packages/ilios-common/addon/models/course.js

```diff
@@ -293,12 +293,12 @@ export default class Course extends Model {
   }
 
   setDatesBasedOnYear() {
-    const today = moment();
-    const firstDayOfYear = moment(this.year + '-7-1', 'YYYY-MM-DD');
+    const today = DateTime.now();
+    const firstDayOfYear = DateTime.fromFormat(this.year + '-07-01', 'yyyy-LL-dd');
```

Comment:
I think the `fromObject` is a bit clearer than `fromFormat`. 

```
const firstDayOfYear = DateTime.fromObject({
  year: this.year,
  month: 7,
  day: 1,
});
```

Reply:
good catch.

replaced :white_check_mark: